### PR TITLE
Updateed reddit.py 2.0

### PR DESCRIPTION
**Pull request by yaba12**
fixed comments

### CODE REVIEW

File: reddit.py [hunk 1]
```diff
@@ -14,45 +18,46 @@
...
-#All require a String parameter that state the subreddit you want.
+	def __init__(self, nameOfSubreddit, limits):
+		self.nameOfSubreddit = nameOfSubreddit
+		self.limits = limits
+		reddit = praw.Reddit(client_id='kCrm1Fz7g6alMw',
+	        	client_secret='z3l8kIhB9zYAQl7QUobTOkBGCnQ',
+	                user_agent='testscriptplswork',
+	                username='joshdunigan',
+	                password='joshdunigan123')
+		subreddit = reddit.subreddit(self.nameOfSubreddit)
```
Comment (Joshuad2uiuc) [inline reddit.py]: should be self.subreddit

File: reddit.py [hunk 2]
```diff
@@ -14,45 +18,46 @@
...
-                user_agent='testscriptplswork',
-                username='joshdunigan',
-                password='joshdunigan123')	
-	return reddit
-#following functions only print from hot and of the top 10. Can change it by changing the "hot" and the limit to a higher or lower number.
-#All require a String parameter that state the subreddit you want.
+	def __init__(self, nameOfSubreddit, limits):
+		self.nameOfSubreddit = nameOfSubreddit
+		self.limits = limits
+		reddit = praw.Reddit(client_id='kCrm1Fz7g6alMw',
```
Comment (Joshuad2uiuc) [inline reddit.py]: should be self.reddit

**Comment by Joshuad2uiuc** [comment]
-1

**Comment by Joshuad2uiuc** [comment]
+1

**Comment by tyang1780** [comment]
+1

**Pull request merged**